test/unit/services/test-get-caseload.js

```diff
@@ -92,54 +77,90 @@ describe('services/get-caseload', function () {
+    caseloadHelper.aggregateTeamTierTotals.withArgs('custodyCaseloads').returns('lduCustodyCaseloads')
+    caseloadHelper.aggregateTeamTierTotals.withArgs('communityCaseloads').returns('lduCommunityCaseloads')
+    caseloadHelper.aggregateTeamTierTotals.withArgs('licenseCaseloads').returns('lduLicenseCaseloads')
 
-  it('should return a results object with all necessary results when org unit is Team', function () {
-    getCaseloadDetail.withArgs(id, teamName).resolves(TEAM_CASELOAD)
-    return getCaseload(id, teamName).then(function (result) {
-      expect(result.caseloadDetails.overallCaseloadDetails).to.eql(OVERALL_CASELOAD)
-      expect(result.caseloadDetails.communityCaseloadDetails).to.eql(CASELOAD)
-      expect(result.caseloadDetails.custodyCaseloadDetails).to.eql(CASELOAD)
-      expect(result.caseloadDetails.licenseCaseloadDetails).to.eql(CASELOAD)
-      expect(result.caseloadDetails.custodyTotalSummary).to.eql(999)
-      expect(result.caseloadDetails.communityTotalSummary).to.eql(999)
-      expect(result.caseloadDetails.licenseTotalSummary).to.eql(999)
+    var expectedCaseloadDetails = {
+      overallCaseloadDetails: 'lduOverallCaseloadDetails',
+      communityCaseloadDetails: 'lduCommunityCaseloads',
+      custodyCaseloadDetails: 'lduCustodyCaseloads',
+      licenseCaseloadDetails: 'lduLicenseCaseloads',
+      overallTotalSummary: 'overallSummary',
+      custodyTotalSummary: 'custodySummary',
+      communityTotalSummary: 'communitySummary',
+      licenseTotalSummary: 'licenseSummary'
+    }
+    return getCaseload(id, lduName)
+    .then(function (result) {
+      expect(caseloadHelper.getCaseloadTierTotalsByTeamByGrade.calledWith(LDU_CASELOAD)).to.be.eql(true)
+      expect(caseloadHelper.getCaseloadSummaryTotalsByTeam.calledWith(LDU_CASELOAD)).to.be.eql(true)
+      expect(caseloadHelper.getCaseloadByType.calledWith(LDU_CASELOAD, 'CUSTODY')).to.be.eql(true)
+      expect(caseloadHelper.getCaseloadByType.calledWith(LDU_CASELOAD, 'COMMUNITY')).to.be.eql(true)
+      expect(caseloadHelper.getCaseloadByType.calledWith(LDU_CASELOAD, 'LICENSE')).to.be.eql(true)
+      expect(caseloadHelper.getCaseloadTotalSummary.calledWith('custodyCaseloads')).to.be.eql(true)
+      expect(caseloadHelper.getCaseloadTotalSummary.calledWith('communityCaseloads')).to.be.eql(true)
+      expect(caseloadHelper.getCaseloadTotalSummary.calledWith('licenseCaseloads')).to.be.eql(true)
+      expect(caseloadHelper.calculateTeamTierPercentages.calledWith('overallCaseloadDetails')).to.be.eql(true)
+      expect(caseloadHelper.aggregateTeamTierTotals.calledWith('custodyCaseloads')).to.be.eql(true)
+      expect(caseloadHelper.aggregateTeamTierTotals.calledWith('communityCaseloads')).to.be.eql(true)
+      expect(caseloadHelper.aggregateTeamTierTotals.calledWith('licenseCaseloads')).to.be.eql(true)
+
+      expect(result.caseloadDetails).to.be.eql(expectedCaseloadDetails)
     })
   })
 })
```

Comment:
Updates to routes and e2e tests for new tables etc?

Reply:
updated e2e tests

Reply:
update route tests